build/client/public/styles/app.css

```diff
@@ -77,21 +77,21 @@ p {
   display: none;
 }
 .alert-error {
-  font-family: 'Maven Pro', sans-serif;
-  font-size: 20px;
+  font-family: 'Source Sans Pro', sans-serif;
```

Comment:
When you change things that are not related to the PR, you should mention it somewhere.